src/components/Events/Events.css.js

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components'; 
+
+export const ContainerEvent = styled.div`
+    position: relative;
+    background: #0F6CDE;
+    margin: 0;
+    font-family: Arial;
+    color: #d4d6d4;
+    `;
+
+export const ColsEvent = styled.div`
+    display: flex;
+    flex-wrap: wrap;
+    padding: 50px;   
+    div{
+    flex: 2;
+    }
+`;
+
+export const H1Event = styled.h1`
+    font-size: 60px;
+    text-transform: uppercase;
+    color: #d4d6d4;
+    font-weight: bold;
+    text-align: left;
+    margin: 0 480px 0 0;
+`;
+
+export const EventsP = styled.p`
+`;
+
+export const Pevent = styled.p`
+    .event  .p-event{
+    font-size: 15px;
+    margin: 5px;
+    }
+    color: #d4d6d4;
+    text-align: left;
+    padding: 0 450px 0 0;
+    font-family: Arial;
+`;
+
+export const Pevent2 = styled.p`
```

Comment:
the name should be more explicit, it is a subtitle or a paragraph
what is the intention for this component?

Reply:
Is a paragraph. The intention is view the events. It is already correct.